ecma402_traits/src/lib.rs

```diff
@@ -48,3 +48,9 @@ pub mod collator;
 ///
 /// The [datetimeformat] mod contains all the needed implementation bits for `Intl.DateTimeFormat`.
 pub mod datetimeformat;
+
+/// A Rust implementation of ECMA 402 RelativeTimeFormat API.
+///
+/// The [relativetimeformat] mod contains all the needed implementation bits for
+/// `Intl.RelativeTimeFormat`.
+pub mod relativetimeformat;
```

Comment:
Nit: I'd use datetime.rs, relative time.rs etc. And only name types DateTimeFormat and RelativeTimeFormat

Reply:
Done.